srcpkgs/gpsim/template

```diff
@@ -1,27 +0,0 @@
-# Template file for 'gpsim'
```

Comment:
Why was this removed?

Reply:
As I said, I was trying to use `gpsim` and `simavr` version provided by Void repo.
By a mistake, I removed `gpsim` entirely instead of reverting to `master`.
TLDR; Fixed